data/ssh-keys/hosts/nfs.ecdsa_key

```diff
@@ -0,0 +1,9 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
I guess that adding the private key is intentional, right?

Reply:
Yes.